src/openforms/contrib/kvk/migrations/0007_kvkconfig_profile_service_kvkconfig_search_service.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 4.2.10 on 2024-03-12 10:10
+
+from django.conf import settings
+from django.core.cache import caches
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def set_kvk_service(apps, _):
+    """
+    Derive the API root for KVK interaction.
+
+    This used to be configured in one Service. Due to possible gateways we have to go for
+    two separate Services since we need to provide different urls. The API root url can
+    be the same for both operations so in this case we use one Service configuration.
+    """
+    KVKConfig = apps.get_model("kvk", "KVKConfig")
+    config = KVKConfig.objects.first()
+    if config is None:
+        return
+
+    if service := config.service:
+        if not (root := service.api_root):
+            return
```

Comment:
```suggestion
    if (service := config.service) and not (root := service.api_root):
        return
```

But I'll not that this code is unsafe (`root` should be marked as possibly unbound below)

At the end of this check, you either have `service` and `service.api_root` both not `None`, or `service` is `None`. In the later case, `root` was never assigned to a value.